src/ui/public/agg_table/agg_table.js

```diff
@@ -95,29 +98,47 @@ uiModules
             formattedColumn.class = 'visualize-table-right';
           }
 
-          const isFieldNumeric = (field && field.type === 'number');
-          const isFirstValueNumeric = _.isNumber(_.get(table, `rows[0][${i}].value`));
+          let fieldType = 'string';
+          const aggType = agg.type;
+          if (aggType && aggType.type === 'metrics') {
+            if (aggType.name === 'top_hits') {
```

Comment:
@ppisljar, no, I'm working on it.  Just have other things right now taking most of my time.